kettle/kettle.py

```diff
@@ -148,6 +154,8 @@ def refresh_choices(self):
 		self.queued_data.append(payload)
 
 	def refresh_options(self):
+		if isinstance(self.game.current_player, PlayerAI):
+			return
```

Comment:
There are no AI clients on the wire, they run locally in fireplace under the GameController. refresh_options() only queues to send to stove.